### DISCUSSION TITLE
Performance createReactManager

### DISCUSSION THREAD

**Discussion by koenvg** [remirror/remirror#851] [Q&A]
Hey,

I noticed that creating the manager `createReactManager` takes a considerable amount of time. From initial testing, it hovers around an 8ms average with Wysiwig example from Storybook. The 8ms becomes a real bottleneck when instantiating multiple editors or anything where you want to have 60 fps for a smooth user experience.

![image](https://user-images.githubusercontent.com/6232861/106179883-e8e3cb00-619b-11eb-9a91-f971643a42f2.png)

I did some initial investigating and found a couple of points where performance could be gained. All of those are all micro-optimizations that might not justify the engineering effort though.

The goal is to start a discussion on how to improve this initial render performance.

**Comment by ifiokjr** [comment]
@koenvg  I'm only just seeing this. 

Thanks for breaking this down. I'm very open to performance improvements and any suggestions are welcome. 

While I haven't focused much on performance concerns to date, more on shaping the API, it's going to be a priority once things are locked down.
